### PR TITLE
Move FormattedText from incubation to release #515

### PR DESCRIPTION
**Pull request by chrhem**
Needed to add FormattedText widget to incubation reactor build because DateChooser has a dependency to it

### CODE REVIEW

File: releng/org.eclipse.nebula.nebula-incubation/pom.xml [hunk 1]
```diff
@@ -38,10 +38,10 @@ Contributors:
 		<module>../org.eclipse.nebula.nebula-parent</module>
 		<module>../../examples/org.eclipse.nebula.examples</module>
 		<module>../../examples/org.eclipse.nebula.examples.feature</module>
+		<module>../../widgets/formattedtext</module>
```
Comment (wimjongman) [inline releng/org.eclipse.nebula.nebula-incubation/pom.xml]: Please move the line back to the original location.

**Review by wimjongman — CHANGES_REQUESTED** [review]
Thanks for the patch @chrhem. 

I think there are some category.xml that sites also have to be changed. Can you take a look at that?

**Comment by chrhem** [comment]
Hi @wimjongman , thanks for the quick review.
category.xml's are updated. PDE tooling changed some whitespaces. If this is unwanted i could revert that.
The line in the incubation pom.xml is back in it's old location.

**Review by wimjongman — APPROVED** [review]

**Comment by wimjongman** [comment]
Thank you!! @chrhem

**Pull request merged**